mithril-signer/src/runtime/signer_services.rs

```diff
@@ -326,12 +327,16 @@ impl<'a> ServiceBuilder for ProductionServiceBuilder<'a> {
             cardano_transactions_builder,
         ));
         let metrics_service = Arc::new(MetricsService::new().unwrap());
+        let preloader_activation = CardanoTransactionsPreloaderActivationAccordingToAggregator::new(
+            certificate_handler.clone(),
```

Comment:
The name should be changed here:
```suggestion
            aggregator_client.clone(),
```